modules/polygon/src/index.js

```diff
@@ -1,5 +1,17 @@
 export {default as Polygon} from './polygon';
 
+export {
+  getPolygonSignedArea,
```

Comment:
Is it necessary to export two versions of everything? The other utilities (cutPath*, cutPolygon*) in this module only deal with flat positions. Since these are new APIs it's better to align the pattern?